packages/core/repl/repl.ts

```diff
@@ -21,6 +28,14 @@ export async function repl(module: Type | DynamicModule) {
     prompt: clc.green('> '),
     ignoreUndefined: true,
   });
+  if (options?.historyPath) {
+    replServer.setupHistory(options.historyPath, (err: any) => {
```

Comment:
```suggestion
    replServer.setupHistory(options.historyPath, (err: Error | null) => {
```

I guess?

![image](https://user-images.githubusercontent.com/13461315/204962561-56ecdf55-ff2c-4b53-a411-69109e6ea8cc.png)